app/views/admin/comments/_comment.html.erb

```diff
@@ -0,0 +1,48 @@
+<div>
+  <article class="crayons-card p-4 pt-3 pb-2 flex flex-col gap-3">
+    <% if comment.commentable %>
+      <header class="flex justify-between gap-4 items-center border-b-1 border-base-10 border-solid border-0 pb-3 -mx-2 px-2">
+        <div class="flex gap-2 items-center">
+          <% if comment.user %>
+            <a href="<%= comment.user.path %>" target="_blank" rel="noopener" class="shrink-0 c-link">
+              <img width="32" height="32" class="radius-full block" src="<%= comment.user.profile_image_url_for(length: 64) %>" alt="<%= comment.user.username %> profile" loading="lazy" />
+            </a>
+          <% end %>
+          <p>
+            <a href="<%= comment.user.path %>" target="_blank" rel="noopener" class="c-link c-link--branded fw-bold"><%= comment.user.username %></a>
+            on:
+            <a href="<%= comment.commentable.path %>" class="c-link c-link--branded"><%= comment.commentable.title %></a>
+          </p>
+        </div>
+      </header>
+    <% end %>
+    <div class="text-styles text-styles--tertiary">
+      <%= sanitize comment.processed_html,
+                   tags: %w[strong em p h1 h2 h3 h4 h5 h6 i u b code pre br ul ol li small sup img a span hr blockquote],
+                   attributes: %w[href strong em ref rel src title alt class] %>
+    </div>
+    <footer class="fs-s flex gap-4 border-t-1 border-base-10 border-solid border-0 pt-2 -mx-2 pl-2 justify-between items-center">
+      <span>❤️ <%= pluralize(comment.public_reactions_count, "like") %></span>
+      <% if comment.reactions.where(user_id: current_user.id).empty? %>
+        <%= form_tag(reactions_path, remote: true) do %>
```

Comment:
There's a script at the bottom of the index view that's supposed to hide the like button after it's clicked, and isn't being included in the standalone page (also note that the script doesn't quite work, the `.inline` toggle is unnecessary and actually prevents the button from being hidden)

Beyond that though, even with the fixed script the behaviour is a bit janky (doesn't update the like count, so it's unclear to the user if something did happen, and you can't unlike the comment). I started on a fix for that (putting in a proper reaction button instead of re-implementing one) but paused because I'm actually not sure if anyone uses these to begin with. We might be better off removing the like functionality entirely - cc @anujbhavsar96. Either way fully dealing with it is probably out of scope for this PR

Reply:
Thanks a lot for all the details.
I am not blocking this PR on this change. Once we get @anujbhavsar96 's reply on this we can deal with this in a separate PR.

Reply:
Oh yes, the Like functionality on that page is a bit buggy. It's a good call to remove the button and display the count. Anyway, there is no use case of Like there.

Reply:
Removed the like button completely. Along with the script code